packages/core/src/Ponder.ts

```diff
@@ -72,8 +74,9 @@ export class Ponder {
     });
     const errors = new UserErrorService();
     const metrics = new MetricsService();
+    const telemetry = new TelemetryService(options);
```

Comment:
I almost always prefer passing an object with named/typed properties over an anonymous positional argument, so this line would read `const telemetry = new TelemetryService({ options} );` using the shorthand, and the `TelemetryService` constructor would be typed `constructor({ options }: { options: Options }) { ... }`.

Reply:
oh good call!